remote-config.jsonc

```
@@ -90,13 +90,13 @@
           "message": "`ddev snapshot -a` will back up all your project databases."
         },
         {
-          "message": "You're invited to the DDEV Advisory Group, bimonthly on first Wednesday"
+          "message": "You're invited to the DDEV Advisory Group, bimonthly on first Wednesday, see the schedule and sign up at https://www.meetup.com/ddev-events/"
```

```suggestion
          "message": "You're invited to the DDEV Advisory Group, first Wednesday every two months, see the schedule and sign up at https://www.meetup.com/ddev-events/"
```

I always get confused with "bimonthly". Happy to go with "every odd-numbered month", "every other month".